src/utils/ValidComponentChildren.js

```diff
@@ -137,11 +135,26 @@ function some(children, func, context) {
   return result;
 }
 
+function toArray(children) {
```

Comment:
does it make sense to leverage `React.Children.toArray` here now that it exists?